### PR TITLE
chore: use pnpm

### PR DESCRIPTION
**Pull request by nachoaldamav**
Use pnpm instead of yarn for dependencies installing.

### CODE REVIEW

**Review by zkochan — APPROVED** [review]

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -75,15 +77,26 @@ jobs:
...
-          node-version: 18
-          check-latest: true
-          cache: yarn
+          node-version: 20
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8.10.0
+          run_install: false
```
Comment (zkochan) [inline .github/workflows/CI.yml:86]:
```suggestion
          run_install: false
          standalone: true
```

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -119,7 +119,7 @@ jobs:
         shell: bash
       - name: Setup node x86
         if: matrix.settings.target == 'i686-pc-windows-msvc'
-        run: pnpx @ialdama/jsonmod --key pnpm.supportedArchitectures.cpu --values ia32
+        run: pnpx @ialdama/jsonmod --key pnpm.supportedArchitectures.cpu --values ia32 current
```
Comment (zkochan) [inline .github/workflows/CI.yml]: interesting. I did not know about this tool.

Comment (nachoaldamav) [reply .github/workflows/CI.yml]: I just created it 😅

**Comment by zkochan** [comment]
Use `pnpm dlx` instead of `pnpx`

**Pull request merged**